### ISSUE TITLE
docker.for.mac.localhost not connecting from inspector

### ISSUE THREAD

**Issue by bcoe** [yukinying/chrome-headless-browser-docker#44]
👋 hello @yukinying thanks for the great work on this container.

Docker for Mac and Docker for Windows resolve the DNS , `docker.for.mac.localhost` and `docker.for.win.localhost` to the host machine's localhost.

These variables would be a very convenient way to run chrome in a headless context, for screenshot testing, without needing to containerize the rest of the local environment.

## Behavior I'm Observing

I'm able to run your container and access `docker.for.mac.localhost` with `wget`, but in an inspector session I get `net::ERR_CONNECTION_REFUSED at http://docker.for.mac.localhost:4000/`.

## Expected Behavior

`http://docker.for.mac.localhost:4000/` would resolve to `127.0.0.1` on the host machine.

**Comment by yukinying** [comment]
I am not very familiar with `docker.for.{mac|win}.localhost`, but this seems to be the case that chrome does not use the same resolver as wget?

**Comment by yukinying** [comment]
Related: 
- https://github.com/SeleniumHQ/docker-selenium/issues/644 
- https://github.com/SeleniumHQ/docker-selenium/issues/511
- https://github.com/docker/for-mac/issues/1837

tl;dr, chrome would resolve *.localhost to 127.0.0.1. And your expected behavior is not correct, in order to resolve correctly (as in your case for wget), the resolved address would be the internal IP that docker assigned to the host (usually it starts with 172.x.x.x) instead of localhost IP (127.0.0.1).

**Comment by yukinying** [comment]
Per https://docs.docker.com/docker-for-mac/networking/, a suggestion is to try those *.internal address.
